frontend/include/chpl/framework/ID.h

```diff
@@ -217,7 +217,7 @@ class ID final {
     return symbolPath_.isEmpty();
```

Comment:
Not really part of the code diff, but you should stop 80col-wrapping your PR description in GitHub. It's harder to read on GitHub itself because it wraps, and GitHub can auto-wrap it to 80 columns when putting into the merge commit's text.

Reply:
Does it really do that now? I guess I could give it a try this PR. But what zoom are you reading GitHub at if it wraps for you?!

Reply:
I'm taking a leap into a brave new world...